packages/e2e-test-utils-playwright/src/request-utils/posts.js

```diff
@@ -3,6 +3,27 @@
  *
  * @this {import('./index').RequestUtils}
  */
+
+export async function createNewPage( title, status = 'draft' ) {
+	const post = await this.rest( {
+		method: 'POST',
+		path: '/wp/v2/pages',
+		params: {
+			title,
+			status,
```

Comment:
Do you think it would be worth linking to the "Statuses" documentation for reference? https://developer.wordpress.org/rest-api/reference/post-statuses/